src/BlazorWasmSdk/Tasks/GenerateBlazorWebAssemblyBootJson.cs

```diff
@@ -85,8 +91,30 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
                 resources = new ResourcesData(),
                 config = new List<string>(),
                 icuDataMode = icuDataMode,
+                startupMemoryCache = ParseOptionalBool(StartupMemoryCache),
             };
 
+            if (!String.IsNullOrEmpty(RuntimeOptions))
+            {
+                string[] runtimeOptions = RuntimeOptions.Split(' ');
+                result.runtimeOptions = runtimeOptions;
+            }
+
+            if (ParseOptionalBool(Jiterpreter) == false) 
```

Comment:
this code assumes that the runtime default is `true`. I would prefer to make it also work in the other direction. 
That is enable it if the runtime default was `false`.